presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -334,6 +338,49 @@ public ConnectorPageSource getPageSource(
         }
     }
 
+    Optional<BitSet> getRowsFromUuid(FileSystem fileSystem, Optional<UUID> deltaShardUuid)
```

Comment:
This should be hidden behind a class (e.g., `DeltaPageSource`), where there is an interface asking the mask with `offset` and `length`.